aspnetcore/blazor/globalization-localization.md

```diff
@@ -873,6 +873,53 @@ To further understand how the Blazor framework processes localization, see the [
 
 :::zone-end
 
+## Shared resources
+
+To create localization shared resources, adopt the following approach. For additional information, see the shared resources guidance in the <xref:fundamentals/localization#make-the-apps-content-localizable> article.
+
+* Create a dummy class with an arbitrary class name. In the following example:
+
+  * The app uses the `BlazorSample` namespace, and localization assets use the `BlazorSample.Localization` namespace.
+  * The dummy class is named `ComponentLocalization`.
+  * The class file is placed in a `Localization` folder at the root of the app.
+
+  `Localization/ComponentLocalization.cs`:
+
+  ```csharp
+  namespace BlazorSample.Localization
+  {
+      public class ComponentLocalization
+      {
+      }
+  }
+  ```
+
+* Create the shared resource files with a **Build Action** of `Embedded resource`. In the following example:
+
+  * The files are placed in the `Localization` folder with the dummy `ComponentLocalization` class (`Localization/ComponentLocalization.cs`).
```

Comment:
We could add a note that `Localization` is resource path that could be set in the `LocalizationOptions`